packages/web/src/features/manage-club/activity-report/frames/ActivityReportDetailFrame.tsx

```diff
@@ -138,7 +138,7 @@ const ActivityReportDetailFrame: React.FC<ActivityReportDetailFrameProps> = ({
               {
                 onSuccess: () => {
                   close();
-                  router.push("/manage-club/activity-report");
+                  router.replace("/manage-club/activity-report");
```

Comment:
얘도 일케 되어야 하나본디? 삭제한 게 목록에 남아잇엉

```suggestion
                  window.location.href = "/manage-club/activity-report";
```

Reply:
어라 아까 됐는데 그냥 우연이었나보군